BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -5,6 +5,9 @@ import Foundation
 /// An enum to represent a feature flag sent by the server
 ///
 enum FeatureFlag: String, Codable {
+    /// Flag to enable/disable the ability to sync TOTP codes with the Authenticator app.
+    case authenticatorSyncEnabled = "authenticator-sync-enabled"
```

Comment:
🤔 Do our other feature flags in general do an "enabled"? Or would just calling the flag `authenticator-sync` in the config response be sufficient?

Reply:
When I checked the other ones, I saw a few with enabled as a suffix.

Reply:
I was also trying to mirror @victor-livefront's [PR in the Authenticator repo](https://github.com/bitwarden/authenticator-ios/pull/140) which has:
```
    case passwordManagerSyncEnabled = "password-manager-sync-enabled"
```